File: output/llama3.1-8b/avatar/Python/Java/codeforces_108_B.java
import java.util.Arrays;
import java.util.Scanner;

public class codeforces_108_B {
    public static void main(String[] args) {
        System.out.println(s());
    }

    public static String s() {
        Scanner scanner = new Scanner(System.in);
        String input1 = scanner.nextLine();
        String[] strArray = scanner.nextLine().split(" ");
        int[] a = new int[strArray.length];
        for (int i = 0; i < strArray.length; i++) {
            a[i] = Integer.parseInt(strArray[i]);
        }
        Arrays.sort(a);
        for (int i = 1; i < a.length; i++) {
            if (a[i] != a[i - 1] && a[i] < a[i - 1] * 2) {
                return "YES";
            }
        }
        return "NO";
    }
}

File: output/llama3.1-8b/avatar/Java/Python/codeforces_108_B.py
import sys
import array

n = int(input())
a = array.array('i', [int(input()) for _ in range(n)])
a.sort()
for i in range(len(a) - 1):
    if a[i + 1] < a[i] * 2 and a[i] != a[i + 1]:
        print("YES")
        break
else:
    print("NO")